### ISSUE TITLE
How do I write into text file or csv file?

### ISSUE THREAD

**Issue by alvinjchoi** [joeyism/linkedin_scraper#7]
Here's what I'm doing...
driver - webdriver.Chrome()
person = Person("http://www.linkedin.com/in/randomperson", driver = driver, scrape=False)
f = open('scrape.txt', 'w')
f.write(person.scrape())
 

But I'm getting an error.. 
TypeError: write() argument must be str, not None
How do I convert person object into string and append into text file, or better yet, csv file?

**Comment by alvinjchoi** [comment]
To give you an idea of what I'm doing, I have around 400 linkedin URLs and want to scrape their name, location, current company, current position, etc.

**Comment by alvinjchoi** [comment]
<img width="769" alt="screen shot 2018-04-08 at 4 02 14 pm" src="https://user-images.githubusercontent.com/31370892/38471940-3a0f05be-3b46-11e8-9291-728e9fb500db.png">

So.. I'm wondering if there's a way to parse through multiple urls and keep scraping.
I'm a very beginner in python, so it's a bit challenging.
I imported time method thinking maybe I'll need some time to give it to load the page.

But here's an error that I'm getting:
<img width="1166" alt="screen shot 2018-04-08 at 4 03 59 pm" src="https://user-images.githubusercontent.com/31370892/38471963-7abec536-3b46-11e8-8314-d14ae6afbaf1.png">

**Comment by joeyism** [comment]
Your error came about because you haven't logged in, and new Linkedin policy requires you to login for a lot of profiles.

You can login first, then loop through your file without closing browser. That way, you don't have to re-login every time

```python
from selenium import webdriver
from linkedin_scraper import Person

driver = webdriver.Chrome()
driver.get("http://www.linkedin.com")

# put a breaker here via input or something
# you must login here

all_users = {}
fp = open("linkedin_url.text", "r")
for line in fp.readlines():
    person = Person(line, driver = driver, scrape=False)
    person.scrape(close_on_complete=False)

    d = person.__dict__.copy()
    del d["driver"]
    d["experiences"] = [ experience.__dict__ for experience in person.experiences]
    d["educations"] = [ education.__dict__ for education in person.educations]
    all_users[person.name] = d #saves it all to one giant dict

import json; json.dump(all_users, open("filename.json", "w+"))
```

As a side note, it'd be a bit easier if you pasted your code in, instead of taking a screenshot.

**Comment by joeyism** [comment]
I updated it with a bugfix, and it's published in `2.1.1`. Update your `linkedin_scraper` and try it again

**Comment by swissbeats93** [comment]
Following all the above steps, this is what results.
![screen shot 2018-07-31 at 11 08 14](https://user-images.githubusercontent.com/30832909/43450040-25c08d06-94b2-11e8-8a3b-208439cd3f9f.png)
